chat-plugins/wifi.js

```diff
@@ -10,9 +10,8 @@ function checkAllAlts(user, list) {
 		if (prevName === user.userid) continue;
 		if (prevName in list) return 'previous name ' + prevName;
 	}
-	var alts = user.getAlts().map(toId);
-	for (var i = 0; i < alts.length; i++) {
-		if (alts[i] in list) return 'alt ' + alts[i];
+	for (var name in list) {
+		if (Users.get(name).latestIp in user.ips) return 'alt ' + name;
```

Comment:
Man, this is basically the same thing.

I mean don't loop comparing IPs. :|

You can do it in O(1) instead of O(n^2) if you just _store_ IPs instead of userids in the first place.